packages/hint/src/lib/cli.ts

```diff
@@ -16,15 +16,20 @@
  */
 import chalk from 'chalk';
 import * as updateNotifier from 'update-notifier';
+import * as inquirer from 'inquirer';
```

Comment:
I removed the dependency from `inquirer` because it was too heavy. You can use [`ask-question`](https://github.com/webhintio/hint/blob/master/packages/hint/src/lib/utils/misc/ask-question.ts) which is the replacement I did.